recipe/patches/comment_out_xdrfile_version.patch

```diff
@@ -0,0 +1,7 @@
+diff --git a/AmberTools/src/cpptraj/src/xdrfile/VERSION b/AmberTools/src/cpptraj/src/xdrfile/VERSION
+index 8926873d..213b6891 100644
+--- a/AmberTools/src/cpptraj/src/xdrfile/VERSION
++++ b/AmberTools/src/cpptraj/src/xdrfile/VERSION
```

Comment:
You need to delete this file. Commenting out and overriding C++20 `version` header will have unintended consequences.